docs/current/outage.md

```diff
@@ -34,9 +34,11 @@ This document outlines vaquero's functionality in outage scenarios. In this anal
 | Outage                       | Effected Vaquero Functionality                                                                                                                                | Acceptable Outage          |
 |:-----------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------|
 | SoT -> VS                    | Vaquero Server cannot update its source of truth, it will assume it has the most recent model.                                                                | Y (potential for stale DM) |
-| VS -> VA                     | The Vaqeuro Agents will not be able to update state or fetch new data models. Agents will continue to act as if they have the most recent model.              | Y (potential for stale DM) |
-| VS Outage                    | If the entire cluster of VS machines are down this is catastrophic, but given at least one VS is running it is acceptable.                                    | Y if entire VS cluster out |
+| VS -> VA                     | The Vaquero Agents will not be able to update state or fetch new data models. Agents will continue to act as if they have the most recent model.              | Y (potential for stale DM) |
```

Comment:
agents will take no knew actions, but can restore booting hosts to their last known state. Will not run validation or shutdown any new boxes